src/server_manager/index.html

```diff
@@ -15,45 +15,40 @@
 -->
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <!--<base href=".">-->
-    <meta name="google" content="notranslate">
-    <meta http-equiv="Content-Language" content="en">
-    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
-    <!-- TODO: remove this once launched to a broader audience -->
-    <meta name="robots" content="noindex,noarchive">
-    <!-- TODO: update this description -->
-    <!-- <meta name="descripton" content="Outline is a browser extension that makes it easy to set up and access proxy services for your trusted friends and family.">
-    <meta property="og:description" content="Outline is a browser extension that makes it easy to set up and access proxy services for your trusted friends and family." /> -->
-
-    <link rel="shortcut icon" href="/ui_components/images/favicon.png">
-    <link rel="stylesheet" href="/ui_components/style.css">
-    <script src="/bower_components/webcomponentsjs/webcomponents-loader.js"></script>
-    <link rel="import" href="/bower_components/app-layout/app-toolbar/app-toolbar.html">
-    <link rel="import" href="/ui_components/app-root.html">
-
-    <!-- Only allow links in the Outline invite and DigitalOcean pages to be included as an iframe -->
-    <meta http-equiv="Content-Security-Policy" content="frame-src ss://* macappstore://* https://s3.amazonaws.com/outline-vpn/ https://cloud.digitalocean.com https://github.com/;">
-    <script src="server_manager/web_app/main.js"></script>
-
-    <style>
-      body {
-        -webkit-font-smoothing: antialiased;
-      }
-
-      /* Show gray background when a dialog is opened */
-      core-overlay-layer.core-opened {
-        background-color: rgba(0, 0, 0, 0.5);
-        width: 100%;
-        height: 100%;
-      }
-    </style>
-
-    <title>Outline Manager</title>
-  </head>
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="Content-Language" content="en">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
+
+  <link rel="shortcut icon" href="/ui_components/images/favicon.png">
+  <link rel="stylesheet" href="/ui_components/style.css">
+  <script src="/bower_components/webcomponentsjs/webcomponents-loader.js"></script>
+  <link rel="import" href="/bower_components/app-layout/app-toolbar/app-toolbar.html">
+  <link rel="import" href="/ui_components/app-root.html">
+  <script src="server_manager/web_app/main.js"></script>
+
+  <!-- TODO: Add Outline servers to connect-src on-demand (in addition to the DigitalOcean API). -->
+  <meta http-equiv="Content-Security-Policy" content="default-src 'unsafe-inline' outline:; connect-src https:; frame-src https://s3.amazonaws.com/outline-vpn/">
```

Comment:
👍what about the other non-http protocols, i.e. macappstore://, ss://?

Reply:
Are you sure? The invitation page links to `macappstore://` for downloading the macOS client and `ss://` for importing the server when the app is already installed.

Reply:
*click* The iframe-d invite? I just tested and:
- Mac App Store opens without mapappstore:// in the CSP
- ss:// link doesn't open with or without macappstore:// in the CSP (in other words, it's broken as-is - I just filed an issue)